api/objects.proto

```diff
@@ -256,8 +256,8 @@ message NetworkAttachment {
 	// List of aliases by which a task is resolved in a network
 	repeated string aliases = 3;
 	
-	// Map of all the driver options for this network
-	map<string,string> driver_opts = 4;
+	// Map of all the driver attachment options for this network
+	map<string,string> driver_attachment_opts = 4;
```

Comment:
This comment really needs to cover the questions I keep asking about this feature:

```
// DriverAttachmentOpts are passed to the network driver upon attaching to the network.
// These are specific to the driver and interpreted according to the semantics of that driver.
//
// Some examples of this include...
//
// Avoid using this field when...
```